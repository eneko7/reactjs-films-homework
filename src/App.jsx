import React from 'react';
import Footer from './components/Footer/Footer.jsx';


class App extends React.Component {
    render() {
        return(
            <div className="main_container">
                <Footer />
            </div>
        );
    }
};

export default App; 