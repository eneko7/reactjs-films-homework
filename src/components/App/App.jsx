import React from 'react';
import './App.scss';
import Footer from '../Footer/Footer.jsx';


class App extends React.Component {
    render() {
        return(
            <div className="main_container">
                <Footer />
            </div>
        );
    }
};

export default App; 