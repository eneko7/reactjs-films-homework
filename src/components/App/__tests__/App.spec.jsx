import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import App from '../index';

describe('App', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<App />);
  it('renders correctly', () => {
    const result = renderer.getRenderOutput();
    expect(result).toMatchSnapshot();
  });
});
