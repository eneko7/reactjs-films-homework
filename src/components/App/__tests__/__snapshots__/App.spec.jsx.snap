// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly 1`] = `
<Switch>
  <Route
    component={[Function]}
    exact={true}
    path="/"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/films/:film"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/films"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/search"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/:sort"
  />
  <Route
    component={[Function]}
    exact={true}
    path="/genre"
  />
</Switch>
`;
